launch/home/etc/services.cfg

```diff
@@ -34,6 +34,6 @@ org.eclipse.smarthome.threadpool:thingHandler=3
 org.eclipse.smarthome.threadpool:discovery=3
 
 # Non-scheduled thread pools can also provide a max size
-org.eclipse.smarthome.threadpool:safeCall=3,10
+org.eclipse.smarthome.threadpool:safeCall=3
```

Comment:
I think 10 would be a better value because that's what is also used when normally launching openHAB:

https://github.com/openhab/openhab-distro/blob/df94f5b3d6bf5bd09dbaeade55ee8e5cea29a2cc/distributions/openhab/src/main/resources/runtime/services.cfg#L14

Reply:
I agree. `3,10` was an old format to say "at least 3, max 10". Now the value is simply the max value, so it should be 10.

Reply:
While we're at it we might also want to sync the other threadpool values from 3 to 5?

https://github.com/openhab/openhab-distro/blob/df94f5b3d6bf5bd09dbaeade55ee8e5cea29a2cc/distributions/openhab/src/main/resources/runtime/services.cfg#L11-L14

Reply:
why not...